### ISSUE TITLE
Live Video not working Version 1.9.2

### ISSUE THREAD

**Issue by parallelandy** [aussieaddons/plugin.video.afl-video#2620]
I have 2 laptops. Works on the older one and not the newer. Have tried to follow steps with similar issues with 1.9.1 however was unable to locate /storage/.kodi/cdm/libwidevinecd.so. Any ideas on how to fix this problem?

**Comment by glennguy** [comment]
Hi @parallelandy 

We need to know a bit more info about your setup.

Can you send us an error report from you non-working setup?


* Turn on debug logging (in settings/system/logging, or press ctrl+shift+d)
* Restart Kodi
* Recreate the issue
* Submit user report from afl-video settings in the debugging tab
* Write back here with the issue number that it gives you

**Comment by parallelandy** [comment]
@glennguy 
Followed your instructions and the error was 4439. I tried to watch [LIVE NOW] AFL TV. I'm assuming watching a live match would give the same error code? Anyway same issue. The cog wheel spins for about 1 minute but nothing happens and it returns to the live game menu. Hope this helps :-)

**Comment by glennguy** [comment]
I think from memory this is caused by an older version of widevinecdm.dll 

Delete C:\Users\andy\AppData\Roaming\Kodi\cdm\widevinecdm.dll and try to watch a live game/AFL.tv and it should prompt to download it which should fix the issue. 

Let me know how it goes and if you need any further help

**Comment by parallelandy** [comment]
@glennguy Unfortunately did this and no luck. I ran the debugging again and got the code 4455.

**Comment by glennguy** [comment]
Doesn't look to be widevine. The video/audio segments are 404ing. The server would only keep a handful of segments available at a time due to being a live stream (from my testing up to around a minute), and more importantly segments that have not yet been published (future segments) would not be available - 404. 

Your logfile 4455 was uploaded at 00:25 (seconds unknown) but your log files says the upload happened at 00:26:48. I've done the math on the times/segment no's on both your log and mine and it seems to be calculated to what our system clocks are set at - so I'm willing to bet that your laptop's clock is running a minute or two fast which is causing your problem.

Try opening the date/time settings in windows and if 'set time automatically' isn't set, turn it on, otherwise toggle it and that should hopefully refresh.

**Comment by parallelandy** [comment]
@glennguy 
This is very very strange. The clock was exactly 1 minute fast and it was already set to automatically update. I tried to force the clock to automatically update however it wouldn't. I manually set the clock to my phone time (which is automatic) and it worked. Unbelievable! I did some further digging and was advised to go into clock setting on the PC, then the 'Internet Time' tab and then synchronise with time.windows.com. Hopefully this keeps the time correct. Thanks for your help. Much appreciated!

4457
